spaces/S000159/properties/P000130.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000159
+property: P000130
+value: true
+---
+ 
+If $x = 0$, then $\{X\}$ is a neighborhood basis at $x$ and {S159|P16}.
+If $0 < x \leq 1$, then $[a,1]$ for $0 < a < x$ form a neighborhood basis at $x$. Any such $[a,1]$ is
+{P202}, namely $a$, so it is {P16}.
```

Comment:
```suggestion
Given a point $x\in X$, every neighborhood of $x$ contains a neighborhood of $x$
of the form $[a,1]$ for some $a\in[0,1]$.
Any such set $[a,1]$ is {P202} with $a$ as focal point, hence {P16}.
```
Possible simplification, so we don't have to distinguish between the cases $x=0$ and $x\ne 0$